src/OpenLoco/Entities/Misc.cpp

```diff
@@ -296,4 +296,21 @@ namespace OpenLoco
             EntityManager::freeEntity(this);
         }
     }
+
+    // 0x00440BBF
+    ExplosionSmoke* ExplosionSmoke::create(const Map::Pos3& loc)
+    {
+        auto t = static_cast<ExplosionSmoke*>(EntityManager::createEntityMisc());
+        if (t != nullptr)
+        {
+            t->var_14 = 44;
```

Comment:
I don't see where these values would come from in ida?

Reply:
![image](https://user-images.githubusercontent.com/1277401/161915605-3d5e264b-9a6a-410b-bd92-d5ba54de211a.png)
you mean these? They are the sprite dimensions var_09 is the spriteWidth, 014 is spriteHeightPositive, 015 spriteHeightNegative they come from the image that is drawn for this entity which will be up to 32 x 78 pixels in size (This is used for invalidation purposes so its okay if its large than the actual size of the image but cannot be less)

Reply:
Guess I'm blind then 😅